test/unit/friends.saga.test.ts

```diff
@@ -124,4 +144,40 @@ describe('Friends sagas', () => {
       })
     })
   })
+
+  describe('get friend requests', () => {
+    beforeEach(() => {
+      const { store } = buildStore()
+      globalThis.globalStore = store
+
+      mockStoreCalls()
+    })
+
+    afterEach(() => {
+      sinon.restore()
+      sinon.reset()
+    })
+
+    describe("When there're sent and received friend requests", () => {
+      it("Should call unity with the declared parameters", () => {
+        const request: GetFriendRequestsPayload = {
+          sentLimit: 10,
+          sentSkip: 0,
```

Comment:
I'd do a test that has a skip and a limit to show that the slice is bringing the expected information as done in the getFriends tests, otherwise, great job!